recipes/clipper2/all/conanfile.py

```diff
@@ -66,7 +66,7 @@ def validate(self):
             )
 
     def source(self):
-        get(self, **self.conan_data["sources"][self.version])
+        get(self, **self.conan_data["sources"][self.version], strip_root = Version(self.version) >= "1.2.3")
```

Comment:
I'm curious, what changed in the new version for this to be needed?

Reply:
@RubenRBS 
I am asking about the release file on this issue. https://github.com/AngusJohnson/Clipper2/issues/670

Reply:
Thanks! Their response confuses me a little, is this then expected?